### ISSUE TITLE
Issues with browsing RO in ontobee

### ISSUE THREAD

**Issue by cmungall** [OntoZoo/ontobee#135]
## Releases are not kept up to date

Last release was may
https://github.com/oborel/obo-relations/releases

ontobee version here is March http://www.ontobee.org/ontology/RO

## Definition source URLs are turned into numbers

Compare
http://purl.obolibrary.org/obo/RO_0002507

Protege:

![image](https://user-images.githubusercontent.com/50745/42470446-513739d4-836f-11e8-9450-cceb1db44a96.png)

In ontobee this just shows up as a number

## inverses only show up in one direction

These are mutual inverses:

http://purl.obolibrary.org/obo/RO_0002508
http://purl.obolibrary.org/obo/RO_0002507

but the inverse only shows for one direction

This is particularly important for RO because we tend not to duplicate redundant info on the inverse. We choose a 'main' direction and then include a pointer

**Comment by hoganwr** [comment]
Just a random thought, but I wonder if the inverse issue has to do with a "manually curated vs. fully-inferenced" version issue.  In other words, is Ontobee sucking in the version that contains all the inference-generated info?  Because if you assert the inverse only in one direction, the reasoner should conclude the inverse in the other, right?

**Comment by cmungall** [comment]
I would not pose this as a reasoning problem. In fact pre-reasoning in OWL would not help, DisjointClasses takes a set as argument, so `Disjoint(A B)` and `Disjoint(B A)` are structurally equivalent as well as semantically.

You could pose this as an RDF-level reasoning problem but I would just state as a programmatic requirement: show disjointFrom in both directions.

**Comment by cmungall** [comment]
btw we will likely incude more inferences in released RO when ROBOT supports, eg domain/range propagated down. But as I say this doesn't help with DisjointClasses, EquivalentProperties and other constructs that takes sets as arguments

**Comment by cmungall** [comment]
Could be done via a simple SPARQL UPDATE rather. This is something that might be generally useful for multiple triplestores, or for ROBOt querying etc, cc @balhoff @jamesaoverton

**Comment by cmungall** [comment]
Looks like @e4ong1031 has addressed the inverseOf issue in the source code: #137

**Comment by e4ong1031** [comment]
@cmungall Yes, the inverseOf was addressed in the pull request #137 . Also, the definition source URL was also addressed in #136 .
